cmd/watch.go

```diff
@@ -2,11 +2,12 @@ package cmd
 
 import (
 	"fmt"
-	"github.com/redhat-developer/odo/pkg/odo/genericclioptions"
 	"net/url"
 	"os"
 	"runtime"
 
+	"github.com/redhat-developer/odo/pkg/odo/genericclioptions"
+
```

Comment:
why empty line here?

Reply:
That's my editor(vscode) added line :laughing: 
I'll fix it